README.md

```diff
@@ -28,3 +28,64 @@ async def main():
 
 asyncio.run(main())
 ```
+
+## Development
+
+Example script for Linux how to run everything.
+
+```sh
+# Checkout the repository
+git clone https://github.com/CorantGmbH/aioairq
+cd aioairq
+
+# Create a virtual environment
+# Hints:
+# - Ensure you have Python 3.9 or above.
+#   If "python3" does not work for you or points to an older "python3.8",
+#   you may need "python3.9" or similar instead to use the correct version.
+# - You might get an error message, if you haven't installed the python3-venv module.
+#   Follow the instructions of the message.
+python3 -m venv .venv
+source .venv/bin/activate
+
+# Install dependencies
+pip install -e .
+
+# You can now create a script (see example above) and run it
+vim hello-air-q.py
+python hello-air-q.py
```

Comment:
How about creating an `examples/` folder with a single (for now) example, e.g.  `query_airq_once.py`, containing the code you are referencing? I suppose at this point it may be a bit more helpful to provide the user with the sample script than to suggest they _paste a snippet in vim_ ;) 

```suggestion
# You can now edit the sample script to your liking, providing the appropriate credentials, 
# then run it using the interpreter from the activated environment
python examples/query_airq_once.py
# For a more convenient way of handling credentials, consider creating a `.env` file as described below.
```

Reply:
A good idea, but TBH I don't want to introduce that here.
The PR has 15 commits already.

I just incorporated the existing example into the new instructions.
For a good example there should already be a nice example of how to not write the credentials into the code. With the current example, there is still a manual step for the user to edit the file. That was my thinking about putting a `vim hello-air-q.py` line into the instructions. I think, for the now this should be sufficient to a user started :-)